src/framework/core/js/ModelTransformationTransforms.js

```diff
@@ -653,12 +695,11 @@ var fluid = fluid || fluid_2_0_0;
         return parsedOffset;
     };
 
-    fluid.transforms.invertArrayIndexation = function (transformSpec, transformer) {
-        var togo = fluid.model.transform.copyInversePaths(transformSpec, transformer);
-        if (!isNaN(Number(togo.offset))) {
-            togo.offset = Number(togo.offset) * (-1);
+    fluid.transforms.invertArrayIndexation = function (transformSpec) {
```

Comment:
Good catch antranig!! I've updated the framework to now always copy the transformSpec object before passing it on to the individual inversion function.

In terms of the multiInput transform, it is actually on purpose that I'm not calling the inversePath copying for all of them. It's being used in linearScale, because this takes as one of the inputs `input` (or `inputPath`) that mimic the behavior of the same variables in the standardInputTransformFunction. But this doesn't hold up for things like the binaryOp transform, which doesn't accept `input` and `inputPath`.

The reason linearScale is a multiInputTransformFunction and not a standardInputTransformFunction is that it uses some extra variables (scale and offset) for which we can get the path equivalent as well as default values.